### PR TITLE
Initial arcade onboarding guidance

### PR DESCRIPTION
**Pull request by markwilkie**
Very minimal so far, but a starting point.

### CODE REVIEW

File: Documentation/ArcadeOnboardingGuidance.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Onboarding onto Arcade
+This is very much a WIP document still.  There are many details below that need to be filled in or clarified.  However, consider this the start.
+
+## Enabling Dependency Flow
+- Take a dependency on Arcade SDK  (this section will likely be broken out in the future versions of this doc)
+  - Add entry in your global.json
+  - Add Directory.Build
+  - Copy 'eng' folder
```
Comment (JohnTortugo) [inline Documentation/ArcadeOnboardingGuidance.md]: Will also need to copy the *.sh/.cmd/.yml/ scripts from the root of the repo, right?

File: Documentation/ArcadeOnboardingGuidance.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Onboarding onto Arcade
+This is very much a WIP document still.  There are many details below that need to be filled in or clarified.  However, consider this the start.
+
+## Enabling Dependency Flow
+- Take a dependency on Arcade SDK  (this section will likely be broken out in the future versions of this doc)
+  - Add entry in your global.json
```
[the remark this answers is not in the document]
Comment (jcagme) [reply Documentation/ArcadeOnboardingGuidance.md]: A reference for the version could be "what ever arcade repo has" meaning, the version included in global.json of the arcade repo

File: Documentation/ArcadeOnboardingGuidance.md (hunk 1 above)
Comment (jcagme) [reply Documentation/ArcadeOnboardingGuidance.md]: No, we don't control those as each repo could add specific logic/files in there

File: Documentation/ArcadeOnboardingGuidance.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  - Add Directory.Build
+  - Copy 'eng' folder
+  - Add the dotnetcore feed to netget.config
+  - Add signtooldata.json
+  - Make sure you have .sln file for repo
+- Onboard official build to VSTS (enables push to BAR)
+  - include base.yml so you have the right hooks
+  - add join phase to yml
+- Create version.details.xml (manual today)
```
Comment (jcagme) [inline Documentation/ArcadeOnboardingGuidance.md]: Version.Details.xml (it matters... :) )

File: Documentation/ArcadeOnboardingGuidance.md (hunk 1 above)
Comment (alexperovich) [reply Documentation/ArcadeOnboardingGuidance.md]: The contents of the `eng` folder should not be copied. Just the `eng\common` folder. The `eng` folder contains repository specific files.

File: Documentation/ArcadeOnboardingGuidance.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Onboarding onto Arcade
+This is very much a WIP document still.  There are many details below that need to be filled in or clarified.  However, consider this the start.
+
+## Enabling Dependency Flow
+- Take a dependency on Arcade SDK  (this section will likely be broken out in the future versions of this doc)
+  - Add entry in your global.json
+  - Add Directory.Build
```
Comment (alexperovich) [inline Documentation/ArcadeOnboardingGuidance.md]: NIT: `Directory.Build.props and Directory.Build.targets`

**Comment by chcosta** [comment]
I hadn't seen this PR until just now.  I wrote up something similar in one of the GitHub issues I'm rewriting to contain some of this data.  The version I wrote up contains links to samples and such, feel free to pilfer the issue if you find the links useful - https://github.com/dotnet/core-eng/issues/3888

I'm waiting until the REST API stuff is in prod before finalizing that issue with links / guidance.

**Comment by mmitche** [comment]
Closing in favor of https://github.com/dotnet/arcade/pull/808

**Pull request closed without merging**